packages/ember-metal/lib/is_empty.ts

```diff
@@ -1,4 +1,3 @@
-import { get } from './property_get';
```

Comment:
This is a separate, unrelated change. Can you keep this PR focused specifically on changing `get` and `getPath` and send another PR for these changes?

Reply:
it is a bit related :P , I did so to avoid warning for calling `isEmpty` on destroyed object in tests